//
// Copyright 2024 Ant Group Co., Ltd.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto3";

package psi.proto;

message KeyInfoMeta {
  bytes keys_hash = 1;
  uint32 duplicate_key_cnt = 2;
  uint32 unique_key_cnt = 3;
  uint32 original_cnt = 4;
  uint64 source_file_size = 5;
}
